pages/linux/lnav.md

```diff
@@ -14,3 +14,7 @@
 - Validate the format of log files against the configuration and report any errors:
 
 `lnav -C {{path/to/log_directory}}`
+
+- View logs of the directory '/var/log':
```

Comment:
I think the first example includes the usage of this. There is no need to add this new example.

Reply:
I'll remove it

Reply:
Yes, I mean this example(line 18~20) is a specific usage of the example one. So we do not accept this  new update.